BoxContentSDK/BoxContentSDK/Models/BOXRecentFile.h

```diff
@@ -0,0 +1,23 @@
+//
+//  BOXRecentFile.h
+//  Pods
+//
+//  Created by Andrew Dempsey on 12/23/16.
+//
+//
+
+#import <BoxContentSDK/BoxContentSDK.h>
+#import "BOXRecentItemProtocol.h"
+
+/**
+ Represents a file recently interacted with by the current user.
+ */
+@interface BOXRecentFile : BOXFile <BOXRecentItemProtocol>
```

Comment:
Why do we need this concrete class and a protocol? It would seem the class would be sufficient (wouldn't support bookmarks, but we already have that problem and the protocol only helps a little).